src/back/TicketManager/tests/PaymentService/TicketManager.PaymentService.DomainTests/Payments/PaymentTests.cs

```diff
@@ -0,0 +1,28 @@
+using FluentAssertions;
```

Comment:
`dotnet add package FluentAssertions --version 6.10.0`

Reply:
Addressed in `17b8c97e`.